packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4432,109 +4592,86 @@ export default class Meeting extends StatelessWebexPlugin {
   }
 
   /**
-   * Monitor the Low-Latency Mercury (LLM) web socket connection on `onError` and `onClose` states
-   * @private
-   * @returns {void}
+   * sets Caption language for the meeting
+   * @param {string} language
+   * @returns {Promise}
    */
-  private monitorTranscriptionSocketConnection() {
-    this.transcription.onCloseSocket((event) => {
-      LoggerProxy.logger.info(
-        `Meeting:index#onCloseSocket -->
-        unable to continue receiving transcription;
-        low-latency mercury web socket connection is closed now.
-        ${event}`
+  public async setCaptionLanguage(language: string) {
+    // wait for the caption language update event as the response is async
+    // check if we can convert this to promise by waiting for event
+    if (!this.isTranscriptionSupported()) {
+      LoggerProxy.logger.error(
+        'Meeting:index#setCaptionLanguage --> Webex Assistant is not supported'
```

Comment:
Why is this log talking about Webex Assistant, but the promise below says Transcription isn't supported? Would this not confuse the developer?

Reply:
I think we should align both and also change them to say, 'Webex Assistant is not enabled/supported'

Will update everywhere